spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-spi/src/main/java/org/springframework/cloud/data/module/ModuleStatus.java

```diff
@@ -60,6 +60,11 @@
 		deployed,
 
 		/**
+		 * The module exited normally
```

Comment:
but can a task be deployed with the equivalent of `module.count > 1` ? May not make sense